instrumentation/instracer/tracer.go

```diff
@@ -98,10 +98,10 @@ func StartSpanWithSpanID(ctx context.Context, name string, spanID uint64, o ...o
 			traceID = sc.TraceID()
 			parentID = sc.SpanID()
 		}
-	} else if traceStr := inslogger.TraceID(ctx); traceStr != "" {
+	} else if traceStr := inslogger.TraceID(ctx); len(traceStr) >= 16 {
```

Comment:
may move this magic number to constant?

Reply:
these constants are hardcoded inside jaeger tracer. So there is no point of having them around